HIRS_Utils/src/main/java/hirs/data/persist/certificate/Certificate.java

```diff
@@ -394,11 +400,16 @@ public Certificate(final byte[] certificateBytes) throws IOException {
                 this.publicKeyModulusHexValue = null;
                 this.publicKeySize = 0;
 
-                authKeyIdentifier = AuthorityKeyIdentifier
-                        .fromExtensions(attCertInfo.getExtensions());
-                this.authorityInfoAccess = getAuthorityInfoAccess(
-                        AuthorityInformationAccess.fromExtensions(
-                                attCertInfo.getExtensions()));
+                Extensions attCertInfoExtensions = attCertInfo.getExtensions();
+                if (attCertInfoExtensions != null) {
+                    authKeyIdentifier = AuthorityKeyIdentifier
+                            .fromExtensions(attCertInfoExtensions);
+                    this.authorityInfoAccess = getAuthorityInfoAccess(
+                            AuthorityInformationAccess.fromExtensions(
+                                    attCertInfoExtensions));
+                } else {
+                    authKeyIdentifier = null;
```

Comment:
The initialization of the temporary value could be set to null and remove the need for an else statement.

Reply:
Totally fair and fixed.